mlserver/rest/app.py

```diff
@@ -101,4 +102,15 @@ def create_app(
         exception_handlers=_EXCEPTION_HANDLERS,  # type: ignore
     )
 
+    if settings.enable_cors:
```

Comment:
nit: should the default be to enable CORS? I'm wondering whether it's the kind of thing that users expect to be enabled by default. Having said so, maybe it makes sense to be restrictive by default. We can always point users to settings if they want to enable it.

Yeah, not sure what's best. What are your thoughts @JakeNeyer ?

Reply:
My gut tells me to leave it disabled by default. That's typically how I've seen other implementations done. I think it would be tricky to have a sensible defaults for Access-Control-Allow-Origin, and without that sensible default, what's really the point?

Reply:
Yuuup, makes complete sense. Let's leave it as it is :+1: